frontend/storybook/src/components/entry/homepage/create-paragraph.ts

```diff
@@ -0,0 +1,73 @@
+import "@elements/entry/home/TOSORT/icon-wparagraph";
+import "@elements/entry/home/sections/create-section";
+import "@elements/core/titles/variants/title-section"
+import "@elements/core/titles/variants/subtitle";
+import "@elements/entry/home/TOSORT/list-type";
+import "@elements/core/buttons/rectangle";
+import "@elements/core/dividers/circle-div"
+import "@elements/entry/home/TOSORT/column-list"
+
+ 
+  
+export default {
+  title: 'Homepage',
+}
+
+
+
+const STR_SUBTITLE = "Learning Through Creation";
+
+const STR_STARTOTITLE = "Create";
+
+const STR_ENDOTITLE = "your own JIGs";
+const STR_PINK = "pink";
+const STR_DARKBLUE = "darkblue";
+ const STR_SIZELARGE = "large";
+
+ const STR_PATHGIRL = "girl@2x.jpg";
+const STR_LINE1="Big content library";
+const STR_LINE2="Smart & friendly interface";
+const STR_LINE3="Teaching through creation"
+const STR_LINE4="All in one";
+const STR_YELLOW="yellow";
+const STR_SMALL="small";
+const STR_MEDIUM="medium";
+const STR_START="Start creating";
+
+
+export const createParagraph = () => {
+    return `
+
+    <create-section>
+    <sub-title size="${STR_MEDIUM}" slot="subtitle" title="${STR_SUBTITLE}"></sub-title>
```

Comment:
The static content should be in elements, not the component.

Reply:
This has been fixed